### PR TITLE
responsive level page

### PR DESCRIPTION
**Pull request by Cyrildab**

### CODE REVIEW

File: frontend/src/components/LevelPage/GameMenu/GameMenu.scss [hunk 1]
```diff
@@ -70,3 +71,16 @@ nav {
...
 }
+
+.hiddenbutton {
+  position: absolute;
+  left: 0;
+  border-radius: 200px;
+  border: 2.5px solid $primaryColor;
+  background-color: $secondaryColor;
+  color: $primaryColor;
+  height: 3vw;
```
Comment (pagouillart) [inline frontend/src/components/LevelPage/GameMenu/GameMenu.scss]: 3vh

**Review by pagouillart — APPROVED** [review]

File: frontend/src/components/LevelPage/GameMenu/GameMenu.scss [hunk 2]
```diff
@@ -38,12 +38,21 @@ nav {
...
-.closeButton {
+.closeNav {
   position: absolute;
+  border: 3px solid $primaryColor;
+  background-color: $secondaryColor;
+  border-radius: 200px;
+  color: white;
+  width: 5vw;
+  font-size: 1vw;
+  height: 3vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/GameMenu.scss]: Avoid vw for height

File: frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss [hunk 3]
```diff
@@ -3,32 +3,33 @@
...
   right: 0;
-  margin-top: 20px;
+  margin-top: 2vw;
   border-radius: 200px;
-  border: 3px solid $primaryColor;
+  border: 0.2vw solid $primaryColor;
   background-color: $secondaryColor;
-  height: 2rem;
-  width: 2rem;
+  height: 2vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss]: Avoid vw for height

File: frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss [hunk 4]
```diff
@@ -3,32 +3,33 @@
...
+  position: absolute;
   display: flex;
   flex-direction: column;
-  position: absolute;
-  right: 30%;
-  bottom: 25%;
   background: $primaryColor;
   color: $secondaryColor;
+  width: 30vw;
+  height: 15vw;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/ReturnGameMenu/ReturnGameMenu.scss]: Avoid vw for height

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**